packages/SwingSet/bin/rekernelize.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env -S node -r esm
```

Comment:
this won't work on linux.
> the Linux kernel treats everything following the first “word” in the shebang line as a single argument.
 -- https://unix.stackexchange.com/questions/399690/multiple-arguments-in-shebang

Reply:
try:

```js
#!/usr/bin/env node

const require = require('esm')(module);
require('node-lmdb');
// ...
const path =require('path');
// ...
```

Reply:
I don't follow this.  Isn't the whole point of `esm` to not require `require`?

Reply:
@warner and I discussed this.  We already have a bunch of other command line utilities that already have the "ESM shebang thing doesn't work on Linux" problem, which the affected parties have dealt with by launching with `node -r esm foo.js ...` directly.  Eventually this will be addressed by endo when it eliminates the need to for the `-r esm` voodoo entirely, so we assess it as not worth investing in fixing this annoyance right now.  (Alternatively, we may just want to produce a little node launcher shell script that can be used in place of `/bin/env`, but that's more of a project too, even if a smallish one.)  So could you please release the request for changes or approve so this can land?  Thanks.

Reply:
yes, but since `-r esm` doesn't fit on the `#!` line, we can't have nice things.

If you want to use `import` syntax, you have to split the tool between two files. :-/

Reply:
or don't bother with a `#!` line and just require the tool to be invoked as `node -r esm rekernelize.js ...`.